test/integration/pool.test.js

```diff
@@ -19,6 +19,10 @@ function assertSameConnection(newConnection, oldConnection) {
       expect(oldConnection.threadId).to.be.equal(newConnection.threadId).and.to.be.ok;
       break;
 
+    case 'db2':
+      expect(newConnection.connected).to.equal(oldConnection.connected).and.to.be.ok;
```

Comment:
Is this comparing booleans? 🙈

Reply:
Ideally we would compare some sort of identifiers here

Reply:
@sdepold Yes, value of newConnection.connected would be a boolean value. Post connection it becomes true.

Reply:
Yeah but this is supposed to verify that the two connections are actually the same I think